examples/gameserver.yaml

```diff
@@ -35,7 +35,7 @@ spec:
   ports:
     # name is a descriptive name for the port
   - name: default
-    # portPolicy has two options:
+    # portPolicy has three options:
     # - "Dynamic" (default) the system allocates a free hostPort for the gameserver, for game clients to connect to
     # - "Static", user defines the hostPort that the game client will connect to. Then onus is on the user to ensure that the
     # - "Passthrough" dynamically sets the `containerPort` to the same value as the dynamically selected hostPort.
```

Comment:
Should we also have a log level example in this file?

Reply:
Yes, definitely. I think we can omit updating fleet.yaml? Or we should update both?